semantic_conventions/resource/container.yaml

```diff
@@ -32,3 +32,25 @@ groups:
         brief: >
           Container image tag.
         examples: ['0.1']
+      - id: image.id
+        type: string
+        brief: >
+          The ID of the container image. Usually a hash algorithm followed by a UUID. The UUID might be abbreviated.
+        examples: ['sha256:f90d814248a0']
```

Comment:
```suggestion
        examples: ['sha256:19c92d0a00d1b66d897bceaa7319bee0dd38a10a851c60bcec9474aa3f01e50f']
```

Reply:
Done in ac909d7f5374ec2a0175d7c3dc3662831cce0cac